compiler/AST/ImportStmt.cpp

```diff
@@ -67,7 +67,8 @@ ImportStmt::ImportStmt(BaseAST* source, const char* rename,
 }
 
 ImportStmt::ImportStmt(BaseAST* source, bool isPrivate,
-                       std::vector<const char*>* namesList):
+                       std::vector<const char*>* namesList,
+                       std::map<const char*, const char*>* renamesList):
```

Comment:
`renamesList`->`renames` ?

Makes it more compatible with the similar variable in UseStmt and this is not a list :)

Reply:
I'd be fine with `names`, instead of `namesList`, too

Reply:
I'm going to call it renamesMap so that it isn't nigh-on indistinguishable from the field name